rfc6962-bis.xml

```diff
@@ -1643,13 +1643,7 @@ corrective action when a misissue is detected.
       </section>
       <section title="Redaction of Public Domain Name Labels">
```

Comment:
I suggest renaming this section "Overly Redacting Domain Name Labels" or "Avoiding Overly Redacting Domain Name Labels".